packages/host/app/components/operator-mode/code-mode.gts

```diff
@@ -395,10 +391,31 @@ export default class CodeMode extends Component<Signature> {
     return this.moduleContentsResource?.declarations || [];
   }
 
+  private get nextSelectedDeclaration() {
+    let nextSelected = this.moduleContentsResource?.declarations.find((dec) => {
+      if (isCardOrFieldDeclaration(dec) && this._nextSelectedCodeRef) {
+        if (dec.exportedAs === this._nextSelectedCodeRef.name) {
+          // this case is needed to handle
+          // - default export  
+          // - renamed export name 
+          return true;
+        }else if(dec.localName === this._nextSelectedCodeRef.name){
+          return true
+        }
+        return false;
+      }
+      return false;
```

Comment:
is there a simpler way to write this condition? the cyclomatic complexity is quite high (3 nested if statements).